services/networking/vpc/controls.yaml

```diff
@@ -25,23 +24,17 @@ controls:
     test_requirements:
       - id: CCC.VPC.C01.TR01
         text: |
-          Verify that default networks are not automatically created upon project initialization.
-        tlp_levels:
-          - tlp_red
-      - id: CCC.VPC.C01.TR02
-        text: |
-          Confirm that only custom networks with appropriate security controls are in place.
+          When a new project/account/subscription is created, then the project/account/subscription MUST NOT contain any default network resources.
```

Comment:
I think that project/account/subscription needs to be replaced with a non-CSP term. I'm unsure out of Azure/GCP/AWS what the terms are, but we could be extending this over time. The account is a trust-boundary - an isolation unit.

Here are a few generic terms you could consider for a trust boundary in CSPs:

* Security Boundary**
* Isolation Unit**
* Resource Container**
* Security Domain**
* Trust Domain**
* Enclave**


* **Tenant**
* **Workspace**

Reply:
I'd prefer this option:

"CSP organization" --> "CSP organization OU" --> **CSP resource container** --> resource(s)

**Why?** 
The above I think covers nearly all clouds and is generic enough to now say project, account, subscription.
Tenant to me is usually a 'user' construct of an AWS Account or GCP project.

Reply:
Resource container makes logical sense for sure - I'm just worried that it's not a recognized term and sounds a bit awkward:

_When a resource container is created, it must not contain any default network resources_

@eddie-knight @damienjburks  - what do you reckon? If it sounds ok, then we can create an issue to create a glossary of terms - may well need that anyway

Reply:
I think "subscription" make more sense to me.

Reply:
@mlysaght2017 — Does Resource Group work here instead? It seems like a generic enough term that there wouldn't be potential for confusion with other things like containers and pub/sub subscriptions.

![image](https://github.com/user-attachments/assets/17303f0d-711d-449e-9d2d-f0166a7f4283)
[ref](https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-setup-guide/organize-resources)